pkg/kubelet/kubelet_getters.go

```diff
@@ -274,6 +274,25 @@ func (kl *Kubelet) getPodVolumePathListFromDisk(podUID types.UID) ([]string, err
 	return volumes, nil
 }
 
+func (kl *Kubelet) getMountedVolumePathListFromDisk(podUID types.UID) ([]string, error) {
+	mountedVolumes := []string{}
+	volumePaths, err := kl.getPodVolumePathListFromDisk(podUID)
+	if err != nil {
+		return volumePaths, err
+	}
+	glog.V(4).Infof("Volume paths exist for pod %s: %v", podUID, volumePaths)
```

Comment:
How often does this function run? Will the V(4) log be too noisy?

Reply:
I remove this log, that is mainly for debugging